tests/data/external/love_ext_script.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python
 # This file is part of ts_salobj.
```

Comment:
This and other files seem to have the wrong license header. I'll send you a list.

Reply:
Solved!